src/App.tsx

```diff
@@ -15,48 +16,115 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Alphabeth = 'alphabet',
+  Length = 'length',
+}
+
+function getPrepearedGoods(
+  goods: string[],
+  sortField: string,
+  isReversed:boolean,
+) {
+  const prepearedGoods = [...goods];
+
+  prepearedGoods.sort((a, b) => {
+    switch (sortField) {
+      case SortType.Alphabeth:
+        return a.localeCompare(b);
+      case SortType.Length:
+        return a.length - b.length;
+      default:
+        return 0;
+    }
+  });
+
```

Comment:
let's avoid sorting when it's not needed
```suggestion
 if(sortField) {
  prepearedGoods.sort((a, b) => {
    switch (sortField) {
      case SortType.Alphabeth:
        return a.localeCompare(b);
      case SortType.Length:
        return a.length - b.length;
      default:
        return 0;
    }
  });
 }
```